apps/docs/components/Card/Card.tsx

```diff
@@ -0,0 +1,10 @@
+import type { ReactNode } from "react";
+import "./card.css";
+
+export function Card({ children }: { children: ReactNode }) {
```

Comment:
For the component use a arrow function and default export 
```suggestion
const Card = ({ children }: { children: ReactNode }) => (
```